Script/PRONTO.py

```diff
@@ -1265,8 +1315,10 @@ def main(argv):
 					if (DNA_run_dir == RNA_run_dir):
 						runID_RNA = runID_DNA
 					else:
-						RNA_run_dir_short = RNA_run_dir.split('/')[-1]
-						runID_RNA = RNA_run_dir_short.split('_TSO')[0]
+						# define capture group for string in front of _TSO_500_LocalApp and use on RNA_run_dir basename (last element in path)
+						tokens = re.search('^(.*)_(?:TSO_500_LocalApp)', os.path.basename(RNA_run_dir))
+						# set runID_RNA to string captured by first group
+						runID_RNA = tokens.group(1)
```

Comment:
unfortunately, this breaks PRONTO for us again. I did not consider that the regex was greedy and always captured most of the string. We had some discussion and testing and maybe this could work:
```suggestion
						# split according to suffix (_TSO_500)_LocalApp_results and use on RNA_run_dir basename (last element in path)
						tokens = re.split('(_TSO_500)?_LocalApp_results', os.path.basename(RNA_run_dir))
						# set runID_RNA to first string in list
						runID_RNA = tokens[0]
```
would you verify that this works?

Reply:
Yes, this works for us. I will update with next commit then. Thanks Martin! :)